exporter/streams.go

```diff
@@ -54,6 +64,18 @@ func getStreamInfo(c redis.Conn, key string) (*streamInfo, error) {
 	return &stream, nil
 }
 
+func getEntryId(redisValue []interface{}, index int) string {
+	if len(redisValue) < index || len(redisValue) > index && len(redisValue[index].([]interface{})) < 2 {
+		return "0-0"
```

Comment:
Why `0-0` as the magic value when you can't extract the EntryId?

Reply:
`0-0` indicates a 0 timestamp and is a valid id for redis streams, for backwards compatibility I thought returning a valid 0 stream value would be okay instead of nil or causing an error.

Reply:
After looking at how it defaults the redis 7 missing metrics when running a redis 6 instance, decided to rather have this return an empty string. I think that might be better then all fields that would be an id are aligned.